### DISCUSSION TITLE
Need help to run this project

### DISCUSSION THREAD

**Discussion by teeshajain17** [enisn/UraniumUI#255] [General]
Hi everyone.  I am new to opensource. I need help to run and understand this project to get started. I would highly appreciate your inputs. 
After cloning this project I am getting multiple build errors. 
![image](https://user-images.githubusercontent.com/87597536/223071165-e4a1fd96-da71-4bfe-ae51-1617aa15ae7e.png)

**Comment by enisn** [comment]
Do you have both .net 6 & 7 SDK on your computer?

Also all the workloads have to be installed: `dotnet workload install maui`

**Reply by teeshajain17** [reply]
Thanks for the reply. 
I installed the workloads.
![image](https://user-images.githubusercontent.com/87597536/223151995-18e5d72b-da50-4d48-904f-b995fd014e66.png)
But still left with four errors. Tried to resolve them but failed.
![image](https://user-images.githubusercontent.com/87597536/223152147-1895317c-268b-463b-b428-f2e5e11728cb.png)

**Reply by enisn** [reply]
It seems the same problem here:
https://stackoverflow.com/questions/74385500/why-does-the-net-maui-application-build-and-run-for-windows-desktop-but-fails-o

**Comment by teeshajain17** [comment]
Thanks, I am able to run. Looking forward to contribute.
